api/src/main/java/com/kodemon/api/dto/UserAuthDTO.java

```diff
@@ -6,6 +6,7 @@
  * @author Matej Poklemba
  */
 public class UserAuthDTO {
```

Comment:
I think, we don't need id here, because there are no crud operations for UserAuth